src/pudl/transform/eiaaeo.py

```diff
@@ -514,6 +514,104 @@ def core_eiaaeo__yearly_projected_generation_in_end_use_sectors_by_fuel_type(
     return renamed_for_pudl
 
 
+@asset(io_manager_key="pudl_io_manager")
+def core_eiaaeo__yearly_projected_energy_use_by_sector_and_type(
+    raw_eiaaeo__energy_consumption_by_sector_and_source,
+):
+    """Projected energy use for commercial, electric power, industrial, residential, and transportation sectors.
+
+    The "Energy Use" series in Table 2 which track figures by sector do not always
+    define each type of usage the same way across sectors. There is detailed
+    information about what is included or excluded in each usage type for each sector
+    in the footnotes of the EIA's online AEO data browser:
+
+      https://www.eia.gov/outlooks/aeo/data/browser/#/?id=2-AEO2023
+
+    The data browser also gives some visibility into the tricky system of subtotals
+    within the Energy Use series. To identify and map subtotal usage types, look for
+    the following features in the data browser display: subtotal series are displayed
+    indented, and include all lines above them which are one level out, up to the
+    next indented line. Delivered Energy and Total are special cases which include
+    those plus all subtotals above. In this way, "Delivered Energy" includes
+    purchased electricity, renewable energy, and an array of fuels based on sector,
+    and explicitly excludes electricity-related losses.
+
+    AEO Energy Use figures are variously referred to as delivered energy, energy
+    consumption, energy use, and energy demand, depending on which usage types are
+    being discussed, and which org and which document is describing them. In PUDL we
+    say energy use or energy consumption.
+    """
+    sanitized = filter_enrich_sanitize(
+        raw_df=raw_eiaaeo__energy_consumption_by_sector_and_source,
+        relevant_series_names=(
+            "Energy Use : Commercial",
+            "Energy Use : Electric Power",
+            "Energy Use : Industrial",
+            "Energy Use : Residential",
+            "Energy Use : Total",
+            "Energy Use : Transportation",
+            "Energy Use : Unspecified",
+        ),
+    ).rename(columns={"subtopic": "dimension"})
+
+    expected_values = {
+        "topic": {"energy_use"},
+        "units": {"quads"},
+        "dimension": {
+            "commercial",
+            "electric_power",
+            "industrial",
+            "residential",
+            "total",
+            "transportation",
+            "unspecified",
+        },
```

Comment:
Curious whether the `total` value shows up in this column as a duplicate or because that value is only reported at the "total" level and not the subcomponents like "commercial", "residential", etc.